### ISSUE TITLE
Volta70 is Not Permitted in KOKKOS_ARCH

### ISSUE THREAD

**Issue by nmhamster** [trilinos/Trilinos#2432]
@trilinos/kokkos 

Using `KOKKOS_ARCH=Volta70` causes an error (see below).

```
Volta70 is not an accepted value for KOKKOS_ARCH.  Please pick from these
choices: None

       AMDAVX
       ARMv80
       ARMv81
       ARMv8-ThunderX
       WSM
       SNB
       HSW
       BDW
       SKX
       KNC
       KNL
       BGQ
       Power7
       Power8
       Power9
       Kepler
       Kepler30
       Kepler32
       Kepler35
       Kepler37
       Maxwell
       Maxwell50
       Maxwell52
       Maxwell53
       Pascal60
       Pascal61
```

**Comment by ibaned** [comment]
@nmhamster This was just fixed in Kokkos develop as kokkos/kokkos#1478 .
Would you like me to patch Trilinos develop ?

**Comment by ibaned** [comment]
Please see #2435

**Comment by ibaned** [comment]
I think this can be closed since #2435 was merged, please comment if something else is still missing.
